airbyte-integrations/connectors/destination-postgres/src/test-integration/java/io/airbyte/integrations/destination/postgres/SshPasswordPostgresDestinationAcceptanceTest.java

```diff
@@ -13,4 +15,31 @@ public SshTunnel.TunnelMethod getTunnelMethod() {
     return SshTunnel.TunnelMethod.SSH_PASSWORD_AUTH;
   }
 
+  @Disabled
+  @Override
+  public void testIncrementalDedupeSync() throws Exception {
```

Comment:
instead of doing both the @Disabled and override with an empty test, can you just do the @Disabled and call the super here anyways ?

Reply:
also, do we know why those tests are failing and if they should be fixed? the @Disabled annotation allows for a comment. Ideally, the comment would point to a github issue

Reply:
I think the functionality is broken and never tested with sshpass tunnel for Dbt normalizations. That functionality will go away with Destinations v2 so ignoring the tests. Same functionality is tested through ssh key based tunnel which works fine.  I'll add message in disabled with the issue we created to track for later.

Reply:
> just do the @disabled and call the super here anyways ?  

I guess. switched to it to see if that correctly disables the test.

Reply:
Done as suggested. Resolving